### ISSUE TITLE
volts publish --token does not work

### ISSUE THREAD

**Issue by sharpSteff** [lapce/lapce-volts#5]
When trying to use volts in CI, I need the --token argument to pass the api key. However I receive an invalid argument error. Also volts publish -h does not show the token argument

**Comment by panekj** [comment]
It's not released yet

**Comment by sharpSteff** [comment]
Yes, that's true. Nevertheless it's an issue. If it's ok, than I can work on this by myself

**Comment by panekj** [comment]
upgrade volts

**Comment by sharpSteff** [comment]
volts still at 0.2.1
https://crates.io/crates/volts
